calico/acl_manager/utils.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2014 Metaswitch Networks
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import traceback
+import os
+import logging
+
+log = logging.getLogger(__name__)
+
+def terminate():  # pragma: nocover
+    """
+    Log an Exception and terminate ACL Manager
+    """
+    log.critical("ACL Manager terminating - exception %s",
+                 traceback.format_exc())
```

Comment:
This also needs a big, big warning that you cannot call this from outside an exception handler.